pkgs/development/tools/conftest/default.nix

```diff
@@ -2,21 +2,27 @@
 
 buildGoModule rec {
   pname = "conftest";
-  version = "0.7.0";
+  version = "0.12.0";
 
   src = fetchFromGitHub {
     owner = "instrumenta";
     repo = "conftest";
     rev = "v${version}";
-    sha256 = "0qq2kp9h91rirlhml5vyzmi7rd4v3pkqjk2bn7mvdn578jnwww24";
+    sha256 = "0blrbbnvnnxyw0idhglqdz16i7g6g86a6kw2iw707bg0yfdl1ncq";
   };
 
+  patches = [
+    # Version 0.12.0 does not build with go 1.13. See https://github.com/instrumenta/conftest/pull/85.
+    # TODO: Remove once https://github.com/instrumenta/conftest/pull/85 is merged and lands in a release.
+    ./go-1.13-deps.patch
```

Comment:
I assume this patch differs from? https://github.com/instrumenta/conftest/commit/e8494d819585452e501728a364e31d38100136de.patch

Reply:
Yes, initially it was the same, but @jpreese asked for more changes, including dependency update. IMO this is too much to add onto the `0.12.0` and can potentially change behavior - the proposed patch is the bare minimum to make the `0.12.0` version build on go 1.13

Reply:
You can see that the proposed patch keeps dependency hashes intact (apart from `xerrors`, since the version dependent on is incompatible with go 1.13)